source/blog/2018-02-28-fedora-atomic-28-feb-18.html.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Fedora 27 Atomic Host February 28th Release
+author: dustymabe
+date: 2018-02-28 00:00:00 UTC
+published: true
+comments: false
+tags: atomic, fedora
+---
+
+Fedora Atomic Host 27.93 is available. We have a new kernel (4.15), [ostree](https://github.com/ostreedev/ostree/releases/tag/v2018.2), and [rpm-ostree](https://github.com/projectatomic/rpm-ostree/releases/tag/v2018.3) in this release.
+
+It is also worth noting that now the `rpm-ostree status` output will prefix the remote:ref with `ostree://` in order to denote the system is following an ostree repository remote (see example below). This is in preparation of some upstream changes related to rpm-ostree jigdo, where updates can be delivered via a special rpm in a yum repo rather than an ostree server/remote.
```

Comment:
So, we're actually in the process of renaming this to `rojig` :). Can we do `s/jigdo/rojig/`? Also maybe a link to the high-level issue: https://github.com/projectatomic/rpm-ostree/issues/1081 ?

Reply:
what about `rpm-ostree jigdo (AKA rojig)` OR `rojig (previously known as rpm-ostree jigdo)` ?

Reply:
Edit it once it's decided and let's get this out - I'll merge when I see a LGTM from @dustymabe.

Reply:
A `git grep` shows we haven't mentioned jigdo at all on the site before right? Let's not even mention it once to reduce search engine noise?